.github/workflows/tests.yml

```diff
@@ -18,7 +18,8 @@ jobs:
           python-version: ${{ matrix.python }}
       - name: Install dependencies
         run: |
-          sudo apt-get -y install miniupnpc gobject-introspection gir1.2-appindicator3-0.1 gir1.2-gsound-1.0 gir1.2-gtk-3.0 \
+          sudo apt update
```

Comment:
Probably same thing here too @mathiascode. `sudo apt upgrade -y`. Although this CI stuff really should be in a separate PR.